### PR TITLE
Replace search status endpoint on event definition page

### PR DESCRIPTION
**Pull request by maxiadlovskii**
## Description
This PR replaces the endpoint for getting the search results. Now we using executing_node_id and job id to get the results 

## Motivation and Context
fix: #20925 

## How Has This Been Tested?
I tested it locally with 2 running nodes and a caddy as a reverse proxy

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/stores/event-definitions/FilterPreviewStore.js [hunk 1]
```diff
@@ -98,7 +106,7 @@ export const FilterPreviewStore = singletonStore(
           resolve(new SearchResult(job));
         } else {
           resolve(delay(250)
-            .then(() => this.jobStatus(job.id))
+            .then(() => this.jobStatus(job.id, job.executing_node))
```
Comment (dennisoelkers) [inline graylog2-web-interface/src/stores/event-definitions/FilterPreviewStore.js:109]: We can replace this with `runPollJob` from `views/stores/SearchJobs` to consolidate this.
Maybe we can even make use of `pollJob` from `executeJobResult` to make this consistent with our general search?

**Review by dennisoelkers — CHANGES_REQUESTED** [review]

File: changelog/unreleased/issue-20925.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Fix failing of executing search on event definituon page"
```
Comment (coffee-squirrel) [inline changelog/unreleased/issue-20925.toml:2]: typo

Comment (maxiadlovskii) [reply changelog/unreleased/issue-20925.toml]: I added runPollJob usage. We can refactor all of this and replace with a search implementation usage. But  I would prefer to do it in separate PR.

File: graylog2-web-interface/src/stores/event-definitions/FilterPreviewStore.js (hunk 1 above)
Comment (maxiadlovskii) [reply graylog2-web-interface/src/stores/event-definitions/FilterPreviewStore.js:101]: I added runPollJob usage. We can refactor all of this and replace with a search implementation usage. But I would prefer to do it in separate PR.

**Pull request merged**